menu.php

```diff
@@ -11,11 +12,30 @@
   <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
 </head>
 
+<?php
+// PHP HEADER
+	include "header.php";
+
+	// change the value of $dbuser and $dbpass to your username and password
+	include 'connectvars.php'; 
```

Comment:
use config not connect vars

Reply:
The connect already occurs in the config file as well so you wont need to cal the connection

Reply:
testing revealed this was not the problem. figured it out though!